FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4587,7 +4591,8 @@ subroutine GFS_physics_driver                                 &
             call get_state("air_temperature_output", t_post_precpd)
             call get_state("specific_humidity_output", qv_post_precpd)
             call get_state("cloud_water_mixing_ratio_output", qc_post_precpd)
-            call get_state("total_precipitation", rain1)
```

Comment:
Not sure I understand how the precip works and want to make sure the land model has what it needs.  I see a diagnostic set right after physics runs, but shouldn't we leave the `call get_state("total_precipitation", rain1)`?  Otherwise we don't have a way of updating with the emulated value.  Let me know if that's not the case.

Reply:
Good catch! I didn't intend to delete this.